### PR TITLE
ci-operator/jobs/cri-o: send ci image build notifications to slack

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-fedora-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

File: ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-periodics.yaml [hunk 1]
```diff
@@ -26,6 +26,17 @@ periodics:
...
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-cri-o-cri-o-main-periodics-setup-fedora-periodic
+  reporter_config:
+    slack:
+      channel: '#forum-node-jira'
+      job_states_to_report:
+      - success
+      - failure
+      - error
+      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
```
Comment (haircommander) [inline ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-periodics.yaml:36]: it's possible success notifications may get noisy, but we can see how we feel after a bit

**Pull request merged**